flag_engine/environments/models.py

```diff
@@ -38,6 +38,7 @@ class EnvironmentModel:
     project: ProjectModel
     feature_states: typing.List[FeatureStateModel] = field(default_factory=list)
     allow_client_traits: bool = True
+    updated_at: datetime = utcnow_with_tz()
```

Comment:
I think you'll need to put this in a `field(default_factory=utcnow_with_tz)` ? 

Otherwise all new Environments will just get the same dt (the one it evaluates on app start up).

Reply:
Ah yes, good catch